src/test/java/marquez/service/models/ServiceModelGenerator.java

```diff
@@ -122,6 +122,14 @@ public static Job newJob(boolean hasDescription) {
         updatedAt);
   }
 
+  public static List<JobRun> newJobRuns(int limit) {
+    return Stream.generate(() -> newJobRun()).limit(limit).collect(toList());
+  }
+
+  public static JobRun newJobRun() {
+    return new JobRun(null, 0, null, null, null, null, null, newTimestamp());
```

Comment:
we should eventually add some generator methods which populate the job run

Reply:
Agreed! The `JobRunResponseMapperTest` only needed a couple parameters to be set, but as we use `newJobRun()` in more and more tests, additional generates like `newJobRunWith(rowUuid)`, `newJobRunWith(RunState)` can be added.